x/mint/types/params_test.go

```diff
@@ -0,0 +1,311 @@
+package types
+
+import (
+	"errors"
+	"github.com/ignite/modules/testutil/sample"
+	"math/rand"
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+)
+
+func TestParamsValidate(t *testing.T) {
```

Comment:
For `Validate` we should have a failing test case for failing verification for params
```
validateMintDenom
validateInflationRateChange
validateInflationMax
validateInflationMin
validateGoalBonded
validateBlocksPerYear
validateDistributionProportions
validateWeightedAddresses
```